src/elements/pearl-chain/pearl-chain-leg/pearl-chain-leg.scss

```diff
@@ -0,0 +1,167 @@
+@use '../../core/styles' as sbb;
+
+@mixin sbb-pearl-chain-dotted {
+  background-color: unset;
+  background-image: linear-gradient(to right, currentcolor 0%, currentcolor 50%, Canvas 50%);
```

Comment:
Could you remove all Canvas or white background usages and replace them with another approach? I know it's not that easy, but I guess it should be possible. (e.g. with mask-image and linear gradient).

Using Canvas prints a white background:
<img width="1043" alt="Bildschirmfoto 2024-11-13 um 14 28 50" src="https://github.com/user-attachments/assets/b264206b-4228-4d77-8dd8-75fbc448a2ec">